apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/SpansResourceTest.java

```diff
@@ -3804,6 +3821,16 @@ private Span getAndAssert(Span expectedSpan, String apiKey, String workspaceName
             assertThat(actualSpan.lastUpdatedAt()).isAfter(expectedSpan.lastUpdatedAt());
             assertThat(actualSpan.createdBy()).isEqualTo(USER);
             assertThat(actualSpan.lastUpdatedBy()).isEqualTo(USER);
+            SoftAssertions.assertSoftly(softly -> {
```

Comment:
There's only one assertion executed in each test run, I don't see why we need a soft assertion here